app/lib/lambdaTrigger.server.ts

```diff
@@ -41,3 +44,9 @@ export async function allSettledOrRaise<T extends readonly unknown[]>(
     }
   ).map(({ value }) => value) as { -readonly [P in keyof T]: Awaited<T[P]> }
 }
+
+export function unmarshallTrigger(
+  item?: Record<string, LambdaTriggerAttributeValue>
+) {
+  return unmarshall(item as Record<string, AttributeValue>)
+}
```

Comment:
This file contains utilities for _all_ kinds of Lambda trigger handlers --- not specifically DynamoDB. Please keep the DynamoDB Table Streams related functionality in the app/table-streams directory.

Reply:
Gotcha, this was a change from when I had multiple files and needed to reuse the function, I'll revert it in the next commit